rfcs/036-holdings-records/README.md

```diff
@@ -0,0 +1,261 @@
+    For example, 853 might contain labels `vol.`, `no.` and `issue`, and 863 might contain the values `1–9`, `1–90`, `2000–2010`.
+    These would be joined to form the string `vol. 1–9 no. 1–90 issue 2000–2010`.
+
+    There are similar pairs 854/864 (for supplementary material) and 855/865 (for captions and indexes).
+
+*   [**856 Electronic Location.**](https://www.oclc.org/bibformats/en/8xx/856.html)
+    See [RFC 035](https://github.com/wellcomecollection/docs/tree/master/rfcs/035-marc-856) about how we model electronic locations for more discussion of this field.
+
+*   **949 Shelfmark.**
+    This contains values like /JOU, /HIS and /MED.
+    These are currently exposed on wellcomelibrary.org, e.g. [b13488284](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1348828?lang=eng)
+
+*   **[843 Reproduction Note](https://help.oclc.org/Metadata_Services/Local_Holdings_Maintenance/OCLC_MARC_local_holdings_format_and_standards/8xx_fields/843_Reproduction_Note).**
+    It's not clear how we're actually using this field: it contains values like "Library Display" and "Reading Room", accompanied by a range of dates, sometimes years passed.
+
+We also have a handful of records using fields 000, 561, 596, 852, 860, 876, 990.
+Because these are so infrequently used, we will skip using them in the initial model.
+
+----
+
+
+
+
+
+
+
+## How should we present holdings records in the Catalogue API?
+
+### Can we present holdings records as Items?
+
+The Catalogue API already has a model for `Work` and `Item`.
+The Work model is populated from bibs, and the `Item` model from items.
+
+We did consider trying to put data from holdings records into the `Item` model, but the data in holdings records doesn't really fit.
+For example:
+
+-   Two items titled *Volume 1* and *Volume 2*, and a holdings record titled *Complete set*.
+    It's useful to know that these two volumes form a complete set, but it wouldn't make sense to store this information on the individual items.
+
+-   Two items titled *Volume 1* and *Volume 2*, and a holdings record titled *Missing volume 3*.
+    Here the holdings record is even more important, but there's no way to encode the absence of an item.
+
+-   A run of journals, with each issue itemised, and a holdings record titled *100 issues, 1957–1972*.
+    The holdings record can provide useful summary information that isn't easily observed from the items.
+
+### Creating a separate Holdings type
+
+Rather than reuse the `Item` type, we will create a new `Holdings` type with the same fields (at least, initially).
+This will appear as a list on a Work called `"holdings"`.
+
+We will populate it as follows:
+
+*   The `title` will come from the concatenated contents of MARC field 866 subfields $a and $z.
+    We will append the label from 853 and 863 (newline-separated), one per 853/863 pair.
+    (We might need to split this out if it becomes unwieldy.)
+
+*   The `locations` list will contain:
```

Comment:
I wonder how this displays in practice. Is every holding location also an item location? If so, can we drop and just have items as the sole way of actually locating something?

Reply:
What do you do on Works that have a holdings but not any items? Presumably we do want location information there?

Reply:
Can we create an unidentified item like we do for Miro?

Reply:
I don't think so.

For some of them, we will, because they have a digital location so we'll create the implicit item.

But we do have some records which:

1. Have a physical location on the holdings
2. Don't have any items
3. Have an enumeration (of volumes/issues) that we can't neatly fit onto the `Item` type

e.g. https://search.wellcomelibrary.org/iii/encore/record/C__Rb1311372?lang=eng

So unfortunately I think we really do need some location data here, even if it's not ideal.

Since a holdings record only has one location, we could reduce this down to `"location"` or `"locationType"`, and guarantee its presence, rather than proliferating ever more lists.

Reply:
That's annoying. Are the locations simpler on holdings? Looks that way from that example? If so, does it even make sense to plug in the full location model? Or can we have strings not things here? It's not particularly useful saying something is on open shelves and then not saying where...

Reply:
The locations tend to be simpler on holdings, yes – but we do get a shelfmark (field 949) and a general location code (e.g. `stax`, `whml`).

Let's see what the data looks like when we actually run through it.

(It looks like something is busted in the reporting cluster, because you can't see fixed field 40.)